tests/compilers/issues.rs

```diff
@@ -214,9 +214,9 @@ fn call_with_static_data_pointers(mut config: crate::Config) -> Result<()> {
         "gas",
         Function::new_native_with_env(&store, env.clone(), gas),
     );
-    let mut imports = ImportObject::new();
-    imports.register("env", exports);
-    let instance = Instance::new(&module, &imports)?;
+    let mut imports = Imports::new();
+    imports.import_namespace("env", exports.get_namespace_externs());
```

Comment:
Can we go back into the `register` nomenclature? (`register_namespace`).
Also, if we could have an `Into` from Exports on that function, it would make the following possible:

```rust
    imports.register_namespace("env", exports);
```

Reply:
Will do!